blue-note/templates/home.html

```diff
@@ -0,0 +1,57 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+
+<!-- wp:group {"tagName":"main","layout":{"type":"constrained","justifyContent":"left"}} -->
+<main class="wp-block-group">
```

Comment:
I think the content of the home page should be wide width instead of full width, it looks a bit weird on very big screens

Reply:
@beafialho if you can confirm this, it would be very helpful, thank you!

Reply:
The header and footer should be full width but the content can be wide.

Reply:
thank you for the feedback @beafialho and thank you for mocking it up too!

![Home (or Front Page)](https://user-images.githubusercontent.com/3593343/224649628-5ab335db-0362-48ee-b90c-2472268b34a6.jpg)